packages/react-storage/src/components/StorageBrowser/Views/Controls/Table.tsx

```diff
@@ -260,13 +377,30 @@ export const LocationsViewTable = (): JSX.Element => {
   ) : (
     <TableControl
       columns={LOCATION_VIEW_COLUMNS}
-      data={data.result}
+      data={tableData}
       renderRowItem={renderRowItem}
+      renderHeaderItem={renderHeaderItem}
     />
   );
 };
 
-export const LocationDetailViewTable = (): JSX.Element => {
+export const LocationDetailViewTable = ({
+  sortFunction,
+}: {
+  sortFunction?: () => LocationItem[];
+}): JSX.Element => {
+  const sortFn = sortFunction ?? defaultTableSort;
```

Comment:
Do you think this would work: 
- `sortFn` - sort function we're going to use
- `customSortFn` - sort function that was passed in
- `defaultSortFn` - default sort function implemented above